drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -29,6 +29,10 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(usb_nrfx);
 
+/* USB device controller access from devicetree */
+#define USBD_NODE DT_INST(0, nordic_nrf_usbd)
```

Comment:
btw, I can not remember exactly but I think the consensus was to use just usb without d, https://github.com/zephyrproject-rtos/zephyr/pull/24362/commits/d7787dff535c99e654f7afa229cb40190af21191#diff-2b6788bec1db035ecf36cf22c40c6a70R46

Reply:
The one's you reference are changed to match the SoC's manual.